xgi/core/hypergraph.py

```diff
@@ -66,11 +67,12 @@ class Hypergraph:
     EdgeView((0, 1, 2, 3))
 
     """
+
     _node_dict_factory = IDDict
     _node_attr_dict_factory = IDDict
-    _hyperedge_dict_factory = IDDict
-    _hyperedge_attr_dict_factory = IDDict
-    _hypergraph_attr_dict_factory = dict
+    _edge_dict_factory = IDDict
+    _edge_attr_dict_factory = IDDict
```

Comment:
what's the reasoning between _hyperedge -> _edge?
i vaguely remember an old conversation where people wanted to keep hyperedge to make it explicit that's it's not pairwise, but might be wrong

Reply:
I changed it so that it matches the convention across the rest of the library e.g., `H.edges`, `edge_fc`, etc.